docs/docs/platform/overview/get-started-with-nhost-cli.md

```diff
@@ -0,0 +1,434 @@
+within a `nhost/` folder.
+
+```
+my-nhost-app/
+  └─ nhost/
+      ├─ config.yaml
+      ├─ emails/
+      ├─ metadata/
+      ├─ migrations/
+      └─ seeds/
+```
+
+Finally, make sure to link your current working directory to your GitHub
+repository:
+
+```bash
+echo "# my-nhost-app" >> README.md
+git init
+git add README.md
+git commit -m "first commit"
+git branch -M main
+git remote add origin https://github.com/[github-username]/my-nhost-app.git
+git push -u origin main
+```
+
+## 2. Start a local development environment
+
+To start a local development environment for your Nhost app, run the following
+command:
+
+```bash
+nhost dev
+```
+
+:::caution
+Make sure [Docker](https://www.docker.com/get-started) is up and running. It’s required for Nhost to work.
+:::
+
+Running this command will start up all the backend services provided by Nhost.
+
+It also runs a web server to serve the Hasura console for the GraphQL engine so
+you can manage the database and try out the API.
+
+The Hasura console should open automatically at
+[http://localhost:1337](http://localhost:1337/).
+
+![Hasura Console](/img/architecture/cli/hasura-console.png)
+
+## 3. Make changes
+
+There are three things the Nhost CLI and the GitHub integration track and apply
+to production:
+
+- Database migrations
+- Hasura Metadata
+- Serverless Functions
+
+### Database migrations
+
+Database changes are tracked and managed through migrations.
+
+:::tip
+It's important that you use the Hasura console to make database changes. Indeed, with the Hasura console, DB migration files are generated incrementally to track changes automatically for you.
+:::
+
+To demonstrate how to make database changes, let's create a new table called
+`messages`, with the following columns:
+
+- `id` (type UUID and default `gen_random_uuid()`),
+- `text` (type Text),
+- `authorId` (type UUID),
+- `createdAt` (type Timestamp and default `now()`)
+
+In the Hasura console, head over to the **data** tab section and click on the
+PostgreSQL database (from the left side navigation) that Nhost provides us.
+
+Click on the **public** schema and the **Create Table** button.
+
+![Create Table](/img/architecture/cli/create-table-step-1.png)
+
+Then, enter the values for creating the `messages` table as mentioned above.
+Also, specify the `id` column as the primary key of the table, and link the
+`authorId` column to the `users.id` column using a foreign key to link the
+`users` and `messages` tables together.
+
+![Create Table](/img/architecture/cli/create-table-step-2.png)
+
+Next, click on the **Add Table** button to create the table.
+
+Finally, check out the `migrations/` folder in your project directory. A
+migration file has been created automatically to reflect our database changes
+and track the new table creation.
+
+The migration was created under `nhost/migrations/default`:
+
+```bash
+$ ls -la nhost/migrations/default
+total 0
+drwxr-xr-x  3 gdangelo  staff   96 Apr 27 17:06 .
```

Comment:
I'd change `gdangelo` to something more generic.

Reply:
fixed